.github/workflows/ci.yml

```diff
@@ -2,26 +2,23 @@ on:
   pull_request:
   push:
     branches:
-      - develop
+      - main
```

Comment:
TODO after this is merged: change main branch to main from https://github.com/City-of-Helsinki/drupal-module-helfi-ahjo-org-chart/settings.